### PR TITLE
Enable flutter luci try by default

### PR DESCRIPTION
**Pull request by keyonghan**
SDK luci try jobs are triggered by labeling `CQ+1`. This PR removes the `CQ+1` label to enable Luci try jobs by default for SDK.

This is part of SDK migration to Luci project.

### CODE REVIEW

File: app_dart/lib/src/request_handlers/github_webhook.dart [hunk 1]
```diff
@@ -23,7 +23,7 @@ import '../request_handling/request_handler.dart';
 import '../service/buildbucket.dart';
 
 /// List of repos that require CQ+1 label.
-const Set<String> kNeedsCQLabelList = <String>{'flutter/flutter'};
+const Set<String> kNeedsCQLabelList = <String>{};
```
Comment (godofredoc) [inline app_dart/lib/src/request_handlers/github_webhook.dart:26]: We won't need the label anymore, would you mind removing it?

Comment (keyonghan) [reply app_dart/lib/src/request_handlers/github_webhook.dart]: label removed, and the corresponding logic/tests.

File: app_dart/lib/src/request_handlers/github_webhook.dart [hunk 2]
```diff
@@ -134,33 +131,13 @@ class GithubWebhook extends RequestHandler<Body> {
         await _scheduleIfMergeable(pullRequestEvent);
         break;
       case 'labeled':
```
Comment (godofredoc) [inline app_dart/lib/src/request_handlers/github_webhook.dart:133]: we still need a break here.

Comment (keyonghan) [reply app_dart/lib/src/request_handlers/github_webhook.dart:133]: Added. Thanks!

**Pull request merged**